ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -111,6 +112,9 @@
         "raw_lines": [
             "use heapsize::HeapSizeOf;",
         ],
+        "flags": [
+            "-ignore-methods",
```

Comment:
Needs https://github.com/servo/rust-bindgen/pull/30

We have to ignore the methods, otherwise it pulls in `already_AddRefed` and a whole bunch of other types.